app/src/main/java/bitmap/transformer/App.java

```diff
@@ -3,17 +3,38 @@
  */
 package bitmap.transformer;
 
-public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
 
-    public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+import java.io.IOException;
+import java.util.Locale;
+
+public class App {
+    public static void main(String[] args) throws IOException {
 
         BitMap newFile= new BitMap(args[0],args[1],args[2]);
+        System.out.println(args[2].toUpperCase());
```

Comment:
Testing what args[] returns? Love seeing things like this in code, gives me an idea of how everyone is thinking.